tools/python/maps_generator/utils/file.py

```diff
@@ -48,6 +62,16 @@ def download_file(url: AnyStr, name: AnyStr, download_if_exists: bool = True):
         logger.info(f"File {name} already exists.")
         return
 
+    if is_file_uri(url):
+        #url uses 'file://' scheme
+        try:
+            copy_overwrite(file_uri_to_path(url), name)
```

Comment:
Is there a `rsync` analog in python to avoid unnecessary copies when files are already in place?

Reply:
There's something called pyrsync.  As [its pypi site describes](https://pypi.org/project/pyrsync/): _It is not a wrapper for rsync, but a set of functions which apply full rsync functionality through Python_. So I think it might be a good equivalent.